Resources/Prototypes/Entities/Objects/Weapons/Bombs/plastic.yml

```
@@ -106,3 +106,49 @@
     maxIntensity: 30
     canCreateVacuum: false
   - type: ExplodeOnTrigger
+
+- type: entity
+  name: cluwne bomb
```

This makes it so people can differentiate between the two in the entity spawn menu.

```suggestion
  name: cluwne bomb
  suffix: Real
```